using_images/s2i_images/php.adoc

```diff
@@ -158,9 +165,18 @@ exec` command] to access the container.
 [[php-hot-deploy]]
 
 == Hot Deploying
-Hot deployment allows you to quickly make and deploy changes to your application
-without having to generate a new S2I build. Hot deployment works in this image
-out of the box.
+Hot deployment allows you to quickly make and deploy changes to your application without having to
+generate a new S2I build. In order to immediately pick up changes made in your application source
+code, you need to run your built image with the `OPCACHE_REVALIDATE_FREQ=0` environment variable.
+For example, see link:../../dev_guide/new_app.html#specifying-environment-variables[the `oc new-app`
+command]. You can use link:../../dev_guide/environment_variables.html#set-environment-variables[the
+`oc env` command] to update environment variables of existing objects.
```

Comment:
I think we should be more explicit here about the function of `OPCACHE_REVALIDATE_FREQ`. What you're saying here suggest that it is used to enable the hot deploy, but it only makes sure the files are reloaded every time, not every 2 seconds (or whatever the default was).

Also, I think you should not include words "the" and "command" in the links, though @adellape might have a different opinion.